azure-pipelines.yml

```diff
@@ -215,3 +215,35 @@ jobs:
       continueOnError: true
       condition: not(succeeded())
 
+- job: macOS_Test
+  pool:
+    vmImage: 'macOS-10.13'
+  strategy:
+    maxParallel: 2
+    matrix:
+      coreclr:
+        _args: --testCoreClr
+        _name: CoreClr
+        _configuration: Debug
```

Comment:
i'd delete the configuration variables here and just inline as there is only one job here.

Reply:
Done